documentation/README.md

```diff
@@ -0,0 +1,82 @@
+# Documentation
+
+The documentation is built using [Docusaurus 2](https://docusaurus.io/). The deployment is done through a centralized build from [IOTA WIKI](https://github.com/iota-community/iota-wiki). To run a local instance the [IOTA WIKI CLI](https://github.com/iota-community/iota-wiki-cli) is used.
+
+## Prerequisites
+
+- [Node.js v14.14+](https://nodejs.org/en/)
+- [yarn](https://yarnpkg.com/getting-started/install)
+
+## Installation
+
+```console
+yarn
+```
+
+This command installs all necessary dependencies.
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Including Markdown or MDX from another file
```

Comment:
So I guess we would remove this and replace it with something like:
```
You can find guidance and an introduction to the needed tools and syntax on [our wiki](https://wiki.iota.org/participate/contribute-to-wiki/for_devs/developer_guide)
```

And there we could potentially also describe the structure below, tools like the cli, diataxis, docusaurus tools and more 🤔